sdk/identity/azure-identity/src/private/token_credential_impl.hpp

```diff
@@ -59,6 +59,31 @@ namespace Azure { namespace Identity { namespace _detail {
         bool asResource,
         bool urlEncode = true);
 
+    /**
+     * @brief Parses JSON that contains access token and its expiration.
+     *
+     * @param jsonString String with a JSON object to parse.
+     * @param accessTokenPropertyName Name of a property in the JSON object that represents access
+     * token.
+     * @param expiresInPropertyName Name of a property in the JSON object that represents token
+     * expiration in number of seconds from now.
+     * @param expiresOnPropertyNames Names of properties in the JSON object that represent token
+     * expiration as absolute date-time stamp. Can be empty, in which case no attempt to parse the
```

Comment:
Do all strings in the vector supposed to represent "expiration as absolute date-time stamp"?

Reply:
Yes, even the vector is called `expiresOnPropertyNames`. I.e. not `expiresInPropertyNames`.
"Expires In" = relative,
"Expires On"/"Expires At"  =absolute.